### ISSUE TITLE
Colour contrast of language toggle button

### ISSUE THREAD

**Issue by stef-barburou** [snap-framework/snap#15]
Language: English

Tool(s): colour contrast analyser

Browser/Screen reader: Firefox and IE

WCAG 2.0 reference: 1.4.3 Contrast (minimum)

Details: AQA found that the contrast fails for the language toggle button both out of focus and in focus.

**Comment by stef-barburou** [comment]
was fixed in a previous commit
